packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -801,12 +801,12 @@ function UpgradeCreditInformation( { product }: { product: ResponseCartProduct }
 	if ( isMonthlyProduct( product ) ) {
 		return (
 			<>
-				{ translate( 'Upgrade Credit: %(upgradeCredit)s applied in first month only', {
+				{ translate( 'Upgrade Credit: %(discount)s applied in first month only', {
 					comment:
 						'The upgrade credit is a pro rated balance of the previous plan which is to be applied' +
 						'as a deduction to the first year of next purchased plan. It will be applied once only in the first term',
 					args: {
-						upgradeCredit: formatCurrency( upgradeCredit, product.currency, {
+						discount: formatCurrency( upgradeCredit, product.currency, {
 							isSmallestUnit: true,
 							stripZeros: true,
 						} ),
```

Comment:
This change was not necessary to fix the issue, but it would be good to have the same variable name in all three texts.